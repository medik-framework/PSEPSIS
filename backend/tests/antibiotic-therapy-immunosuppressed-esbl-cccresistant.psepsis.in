[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AgeEntered",
    "eventArgs": [500]
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "WeightEntered",
    "eventArgs": [14],
    "waitFor": {
      "name": "ConfirmAgeEntered",
      "args": []
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartAntibioticTherapy",
    "eventArgs": [],
    "waitFor": {
      "name": "ConfirmWeightEntered",
      "args": []
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AntibioticsPrimaryConditionsEntered",
    "eventArgs": [true, false, false, false, true, false],
    "waitFor": {
      "name": "Instruct",
      "args": ["get antibiotics primary conditions"]
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "CefepimeCeftazidimeCarbapenamResistanceEntered",
    "eventArgs": [],
    "waitFor": {
      "name": "RecommendDrug"
    }
  },
  {
    "action":"exit",
    "waitFor": {
      "name": "Recommend"
    }
  }
]
