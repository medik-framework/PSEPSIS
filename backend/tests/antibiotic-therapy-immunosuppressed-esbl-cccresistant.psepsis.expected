[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmWeightEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get immunosuppression state"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get pseudomonas risk"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get penicillin tolerance"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get recent broad spectrum antibiotics history"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get fungal infection risk"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get rickettsial risk"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get ESBL prevalence"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get Cefepime/Ceftadizime/Carbapenam resistance"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get Staphylococcus prevalence"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get HSV risk"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "RecommendDrug",
    "args": [
        "Children > 28 days who are immunosuppressed or at risk for infection with Pseudomonas species",
        "OSF Antibiotics Set",
        "drug combination",
        "vancomycin (15 mg/kg, maximum 1 to 2 g, for the initial dose)"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Recommend",
    "args": [
        "consider additional antibiotics",
        "PLUS cefepime (50 mg/kg, maximum 2 g, for the initial dose) OR ceftadizime (50 mg/kg, maximium 2 g, for the initial dose) OR carbapenam (eg, imipenem, meroenem) in settings where bacterial organisms with extended-specturm beta-lactamase (ESBL) resistance are prevalent or for patients who have been recently (within two weeks) treated with broad-spectrum antibiotics (eg, third-generation celphalosporin, or fluoroquinolone)"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Recommend",
    "args": [
        "consider additional antibiotics",
        "ADD an Aminoglycoside (e.g. gentamicin, amikacin) for cefepime/ceftadizime/carbapenam resistance"
    ]
  }
]

